### PR TITLE
CY-384 Add docs about kill-cancellation

### PR DESCRIPTION
**Pull request by tehasdf**

### CODE REVIEW

File: content/working_with/workflows/cancelling-execution.md [hunk 1]
```diff
@@ -19,27 +19,30 @@ There are three types of execution cancellations:
...
+    Only use kill cancellation when:
 
+    * The execution engine is unresponsive and other cancellation methods have no effect
+    * After the workflow itself has been cancelled using one of the other methods and there are some operations still running
 
+    In these scenarios you can use kill cancellation on executions that are already in the `cancelled` state. It stops any operations of that execution that are left running and has no effect otherwise.
+    The processes are stopped by sending the SIGTERM signal on Linux (or the TerminateProcess Win32 API on windows), and then SIGKILL about 5 seconds later (or the same TerminateProcess call on Windows).
 {{% warning title="Warning" %}}
-When the execution's status changes to `cancelled`, it means the workflow execution has completed, meaning no new tasks will be started; However, tasks that have already been started might still be executing on agents. This is true for both Standard and Forced cancellations (but not for Kill-cancellations).
+Using the kill-cancellation method means that workflow and operation processes might not have any opportunity to clean up or release resources.
```
Comment (tehasdf) [inline content/working_with/workflows/cancelling-execution.md]: @bmansheim should we drop the dash `-` for consistency?

File: content/working_with/workflows/cancelling-execution.md [hunk 2]
```diff
@@ -19,27 +19,30 @@ There are three types of execution cancellations:
...
 {{% /warning %}}
 
+
 {{% warning title="Warning" %}}
-Using the kill-cancellation method means that workflow and operation processes might not have any opportunity to clean up or release resources.
+When the execution's status changes to `cancelled`, it means the workflow execution has completed, meaning no new tasks will be started; However, tasks that have already been started might still be executing on agents. This is true for both Standard and Forced cancellations (but not for Kill-cancellations).
```
Comment (tehasdf) [inline content/working_with/workflows/cancelling-execution.md]: ditto dash

**Comment by bmansheim** [comment]
Mistakenly rebased to master

**Comment by bmansheim** [comment]
Merged from executions-kill-cancel-fix

**Pull request closed without merging**